### DISCUSSION TITLE
SS stopped développement ?

### DISCUSSION THREAD

**Discussion by xTremePower** [supermerill/SuperSlicer#849] [Q&A]
Hello, I really love ss, meany cool features .
But since the PS export version the was no updates.
And it's say " merged with ps....."

I really don't want to go back to PS, many features will miss me. Like the single wall top layer and

Is SS continue the development?
. thanks a lot for your work 🥰

**Comment by supermerill** [comment]
Yes! euh... no!
Sorry for the hiatus.
